### PR TITLE
Added image and math support to the table plugin

### PR DESCRIPTION
**Pull request by lakatosandrei**
Fixes these 2 issues: [1](https://github.com/pie-framework/pie-elements/issues/73) and [2](https://github.com/pie-framework/pie-elements/issues/74).

### CODE REVIEW

File: packages/editable-html/src/plugins/table/index.jsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ TableCell.propTypes = {
   ]).isRequired
 };
 
-export default opts => {
+export default (opts, imagePlugin, mathPlugin) => {
```
Comment (edeustace) [inline packages/editable-html/src/plugins/table/index.jsx]: The only thing the `TablePlugin` needs is the plugins array, it doesn't need the `imagePlugin` and `mathPlugin` explicitly. To that end: 
```suggestion
export default (opts, toolbarPlugins /* :  {toolbar: {}}[] */) => {
```

File: packages/editable-html/src/plugins/table/index.jsx [hunk 2]
```diff
@@ -158,6 +158,10 @@ export default opts => {
 
       const Tb = () => (
         <TableToolbar
+          imagePlugin={imagePlugin}
```
Comment (edeustace) [inline packages/editable-html/src/plugins/table/index.jsx]:
```suggestion
          plugins={toolbaPlugins}
```

File: packages/editable-html/src/plugins/table/table-toolbar.jsx [hunk 3]
```diff
@@ -66,6 +75,22 @@ export class TableToolbar extends React.Component {
           <Button onClick={onRemoveTable}>
             <RemoveTable />
           </Button>
+          {
```
Comment (edeustace) [inline packages/editable-html/src/plugins/table/table-toolbar.jsx]:
```suggestion
          { plugins.map( p => <ToolbarButton {...p.toolbar} value={value} onChange={onChange} /> ) }
```

**Comment by lakatosandrei** [comment]
@edeustace 
Made the requested changes and fixed the tests.

**Pull request merged**